tests/modules/nf-core/ashlar/zero_uuid.nf

```diff
@@ -0,0 +1,21 @@
+nextflow.enable.dsl=2
+
+process ZERO_UUID {
+
+    input:
+    val(file_in)
+    val(offset)
+
+    when:
+    file_in != "versions.yml"
+
+    script:
+    def file_path = file_in[1]
+
+    """
+
+    echo -n "00000000-0000-0000-0000-000000000000" | dd of=$file_path bs=1 seek=$offset conv=notrunc
```

Comment:
Yes, this modifies the output tif from ashlar in place.  We didn't think this would be a problem because this is just in the test code.  When will this be a problem?  Is it because of the parallelism?

If I'm not operating on the output file from ashlar I think we're back to the same problem we're trying to work around in that the UUID varies for each of the output files from ashlar and we don't get a consistent md5sum when we test.  Am I not understanding a way to solve this?

Is it okay to have this workaround until ashlar is updated to accept a UUID as an input parameter?

Reply:
Apologies, I think I misunderstood your previous question.  Does 'original file' above refer to the input file to ashlar?

The zero_uuid process is not operating on the input file to the ashlar module.  It is operating on the output file from the ashlar module.  It's in place in that `dd` does not generate a new output file but instead overwrites the UUID in the output file from ashlar.

Reply:
Oh, I went back to the workflow documentation and I think I see what you're saying.  I'll try to modify the code and I'll let you know if I have problems.  Thanks!!

Reply:
Oh, I was looking at adding a local variable which seems to work, but in reading over your reply again it sounds like just having `path(tif)` is sufficient.  Is this correct?

Reply:
Hi @adamrtalbot, it sounds like changing `val(tif)` to `path(tif)` in the ZERO_UUID process is all we need for this to work correctly.  Is that correct?  Thanks!